packages/datagateway-dataview/src/views/table/dls/dlsDatafilesTable.component.test.tsx

```diff
@@ -40,6 +39,7 @@ describe('DLS datafiles table component', () => {
         dgcommon: dGCommonInitialState,
       })
     );
+    // TODO: Missing Download properties?
```

Comment:
Looking at the possible types for `dgcommon.data` it seems the test data has more in common with the `ICATEntity` than the `Download` entity, it just seems to be missing a `CREATE_TIME` entry. The tests are currently passing so it doesn't seem like this is problem as is, but it might be worth adding that in here (and the other tests) to surpress the warning.

Reply:
@patrick-austin this was a warning that I had when doing tests. It didn't affect how the tests run but was more of a visual warning. It is mainly due to the fact that `ICATEntity` encompasses all the other types since this was required for how the card view would deal with different types of data.